src/tracks/playabletrack/wavetrack/ui/SpectrumView.cpp

```diff
@@ -609,7 +609,11 @@ void DrawClipSpectrum(TrackPanelDrawingContext &context,
 
```

Comment:
What are the "clip frame drawing fixes?"  Corrections to the previous commit, or some intentional changes specified in the design?  If the first, squash with other commits for wave and note track affordances once those are separated.  If the second, explain more in commit comments.